plugins/KMZ-Map/src/includes/initialize.php

```diff
@@ -0,0 +1,55 @@
+<?php
+// Error Reporting
+error_reporting(E_ALL); // Reports all errors
+ini_set('display_errors','Off'); // Do not display errors for the end-users (security issue)
+ini_set('error_log', PROJECT_PATH.'/data/plugin.log'); // Set a logging file
+file_put_contents(PROJECT_PATH.'/data/plugin.log', '', LOCK_EX);
+
+// ## Setup Environment Constants
+$ucrm_string = file_get_contents(PROJECT_PATH."/ucrm.json");
+$ucrm_json = json_decode($ucrm_string);
+define("UCRM_PUBLIC_URL", $ucrm_json->ucrmPublicUrl);
+
+$config_path = PROJECT_PATH."/data/config.json";
+
+// ## Setup user configuration settings, if they exist
+if (file_exists($config_path)) {
+
+  $config_string = file_get_contents($config_path);
+  $config_json = json_decode($config_string);
+
+  define("API_KEY", $config_json->requiredGoogleApiKey);
+  define("KMZ_FILE", $config_json->requiredKMZFile);
+
+  if (!empty($config_json->optionalLogoUrl)) {
+    define("LOGO_URL", $config_json->optionalLogoUrl);
+  } else {
+    define("LOGO_URL", null);
+  }
+  if (!empty($config_json->optionalFormDescription)) {
+    define("FORM_DESCRIPTION", $config_json->optionalFormDescription);
+  } else {
+    define("FORM_DESCRIPTION", null);
+  }
+  if (!empty($config_json->optionalLinkOne)) {
+    $link_one_string = $config_json->optionalLinkOne;
+    $link_array = explode('|', $link_one_string);
```

Comment:
1 more thing, since you only ever want 2 elements from explode, use the limit argument. I would maybe switch around the order, first link, then text. In that way the text could actually contain the `|` character.

```php
[$link1, $text1] = explode('|', $link_one_string, 2);
define("LINK_ONE", $link1);
define("TEXT_ONE", $text1);